content/en/docs/what-is-opentelemetry.md

```diff
@@ -1,52 +1,60 @@
 ---
 title: What is OpenTelemetry?
-description: A short explanation of what OpenTelemetry is, and is not.
+description: A short explanation of what OpenTelemetry is and isn't.
 aliases: [/about, /docs/concepts/what-is-opentelemetry, /otel]
 weight: -1
 ---
 
 OpenTelemetry is an
 [Observability](/docs/concepts/observability-primer/#what-is-observability)
-framework and toolkit designed to create and manage _telemetry data_ such as
+framework and toolkit designed to create and manage telemetry data such as
 [traces](/docs/concepts/signals/traces/),
 [metrics](/docs/concepts/signals/metrics/), and
 [logs](/docs/concepts/signals/logs/). Crucially, OpenTelemetry is vendor- and
 tool-agnostic, meaning that it can be used with a broad variety of Observability
-backends, including open source tools like
+back ends, including open source tools like
 [Jaeger](https://www.jaegertracing.io/) and
 [Prometheus](https://prometheus.io/), as well as commercial offerings.
-OpenTelemetry is a
-[Cloud Native Computing Foundation (CNCF)](https://www.cncf.io) project.
 
-## Why OpenTelemetry?
+OpenTelemetry is not an observability back end like Jaeger, Prometheus, or other
+commercial vendors. OpenTelemetry is focused on the generation, collection,
+management, and export of telemetry data. The storage and visualization of that
+data is intentionally left to other tools.
+
+## What is observability?
 
-With the rise of cloud computing, microservices architectures, and ever-more
-complex business requirements, the need for
-[Observability](/docs/concepts/observability-primer/#what-is-observability) has
-never been greater. Observability is the ability to understand the internal
-state of a system by examining its outputs. In the context of software, this
-means being able to understand the internal state of a system by examining its
-telemetry data, which includes traces, metrics, and logs.
+[Observability](/docs/concepts/observability-primer/#what-is-observability) is
+the ability to understand the internal state of a system by examining its
+outputs. In the context of software, this means being able to understand the
+internal state of a system by examining its telemetry data, which includes
+traces, metrics, and logs.
 
-In order to make a system observable, it must be instrumented. That is, the code
-must emit [traces](/docs/concepts/observability-primer/#distributed-traces),
+To make a system observable, it must be instrumented. That is, the code must
+emit [traces](/docs/concepts/observability-primer/#distributed-traces),
 [metrics](/docs/concepts/observability-primer/#reliability--metrics), and
 [logs](/docs/concepts/observability-primer/#logs). The instrumented data must
```

Comment:
I wonder if we should link to the signal pages and not to the chapters within the observability primer? Also we could put a link to https://opentelemetry.io/docs/concepts/instrumentation/ when the word instrumented is mentioned first.